getting_started/workflow/export/android_custom_build.rst

```diff
@@ -84,17 +84,15 @@ an empty directory). On Windows, the following path is usually good enough:
     If you already have an android-sdk folder, normally located in ``%LOCALAPPDATA%\Android\Sdk``, 
     then use this folder instead of creating an empty ``android-sdk`` folder. 
 
-Unzip the Android SDK ZIP file into the ``android-sdk`` folder. This folder should 
-now contain the unzipped folder called ``tools``. Rename ``tools`` to ``latest``. 
-Finally, create an empty folder named ``cmdline-tools`` and place ``latest`` into it. 
+Unzip the Android SDK ZIP file into ``andriod-sdk`` folder. This folder should
+now contain the unzipped folder called ``tools``.
```

Comment:
The `tools` directory was also deprecated. See https://github.com/godotengine/godot/pull/38172